packages/grafana-ui/src/components/ColorPicker/NamedColorsGroup.tsx

```diff
@@ -46,6 +47,7 @@ const NamedColorsGroup: FunctionComponent<NamedColorsGroupProps> = ({
                   isSelected={shade.name === selectedColor}
                   color={shade.color}
                   onClick={() => onColorSelect(shade.name)}
+                  onKeyDown={() => onColorSelect(primaryShade.name)}
```

Comment:
Should this be
```suggestion
                  onKeyDown={() => onColorSelect(shade.name)}
```

Reply:
I think the current implementation has a bug because of the above 😉

Reply:
> With the current implementation any keydown event will cause the NamedColorsGroup to change color, not sure that is something we want.

True...would push a fix now

Reply:
> Also, what are your thoughts on the user experience? If I want a Purple:ish color I've to tab a lot of times to get there.

Yeah...I was actually considering implementing just tabs within this current scope of work. Definitely there's some work to be done UX-wise (e.g adding arrow navigation) but I'm considering doing that in a separate PR.

Reply:
Great catch! Thanks!

Reply:
Are you aware that you can commit suggestions directly in the Github UI?